components/plan-build/bin/hab-plan-build.sh

```diff
@@ -2536,6 +2555,8 @@ _resolve_dependencies
 
 _set_path
 
+do_automatic
```

Comment:
I'd argue that we want to make this an internal/private function only and not a user-facing part of our build API. Naming this `_do_automatic` or even possibly `_load_automatic` would be okay. (Sorry about the repeat here, I realized this at the end and circled back to the top)